bundles/org.openhab.binding.iammeter/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="iammeter" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Iammeter Binding</name>
+	<description>This is the binding for Iammeter.</description>
```

Comment:
Can you be a bit more specific, that somebody who isn't aware of Iammeter gets a clue what this is about?

Reply:
edited.